dotCMS/src/main/java/com/dotcms/rest/api/v1/page/PageResource.java

```diff
@@ -246,7 +246,7 @@ public Response renderHTMLOnly(@Context final HttpServletRequest request,
                     (HTMLPageAsset) APILocator.getHTMLPageAssetAPI().findPage(uri, user, mode.respectAnonPerms) :
                     this.pageResourceHelper.getPage(request, user, uri, mode);
 
-            final Template template = this.templateAPI.findWorkingTemplate(page.getTemplateId(), user, false);
+            final Template template = this.templateAPI.findWorkingTemplate(page.getTemplateId(), APILocator.getUserAPI().getSystemUser(), false);
```

Comment:
Why should we use the system user here?, why not to use the logged in user, templates are permissionable, not sure the logic behind this but only a user with the right permissions should be able to render a template right?

Reply:
Looks like that is how it used to work.